README.md

```diff
@@ -1,4 +1,15 @@
 # ThavalonWeb
-Web App for THavalon
+This repo contains the front-end web application, back-end API, and database code for ThavalonWeb. For a description of Thavalon, see below. To view the rules, see the [rules page](docs/rules.md).
 
-Starting over.
+# What is Thavalon?
+THavalon is a massive extension of the rules presented in Don Eskridge's social deception game, The Resistance: Avalon. Thavalon was designed over several years primarily by [Andrew Hitt](https://github.com/aquadrizzt/THavalon). The main point of THavalon is to provide every player of the game with a role and that to ensure every player in the game feels like they can make an impact on the result of the game. Over time, these rules have evolved in an effort to make the game both faster and more fun, as well as to fix some of the playstyles that have been established in my main testing group's metagame.
```

Comment:
should probably link to user here, not repo

Reply:
Fixed.